tests/python/pants_test/pantsd/test_pantsd_integration.py

```diff
@@ -22,6 +22,7 @@
 def launch_file_toucher(f):
     """Launch a loop to touch the given file, and return a function to call to stop and join it."""
     if not os.path.isfile(f):
+
```

Comment:
Not sure if this was intentional.

Reply:
Guess which file I was smoke testing with, heh.